### PR TITLE
[improvement](regression-test) add partial update with schema change case

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

add partial update with schema change case.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by freemandealer — APPROVED** [review]
LGTM

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_schema_change.groovy [hunk 1]
```diff
@@ -17,39 +17,1093 @@
...
+            }
+            log.info("Stream load result: ${result}".toString())
+            def json = parseJson(result)
+            assertEquals("success", json.Status.toLowerCase())
+            assertEquals(1, json.NumberTotalRows)
+            assertEquals(0, json.NumberFilteredRows)
+            assertEquals(0, json.NumberUnselectedRows)
+        }
+    }
+    qt_sql1 " select * from ${tableName}"
```
Comment (yiguolei) [inline regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_schema_change.groovy:63]: add order by please

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**